deploy/role.yaml

```diff
@@ -116,13 +116,23 @@ rules:
       - namespaces
       - namespaces/finalizers
       - serviceaccounts
+      - configmaps
     verbs:
       - list
       - create
       - watch
       - get
       - update
       - delete
+  # This is used to allow the operator to setup catalog source and remove it if needed.
+  - apiGroups:
+      - operators.coreos.com
+    resources:
+      - catalogsources
+    verbs:
+      - create
+      - get
+      - update
```

Comment:
Do you also need a `delete` here if it needs to be removed?

Reply:
Doesn't look to require it since the `CatalogSource` gets created in the product namespace and gets removed when the the namespace is deleted unlike previously where a `CatalogSourceConfig` would get created / deleted from the the `openshift-marketplace` namespace